.travis.yml

```diff
@@ -11,6 +11,8 @@ env:
   - SYMFONY_VERSION=2.5.*
 
 matrix:
+  allow_failures:
+    - php: hhvm
```

Comment:
Not sure, but I have the feeling that hhvm is unstable and is always being a pain.